app/views/katello/layouts/react.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for(:javascripts) do %>
+  <%= javascript_include_tag *webpack_asset_paths('katello', :extension => 'js'), "data-turbolinks-track" => true %>
```

Comment:
I wonder if we should add this path also to `katello.html.erb` layout to unlock using new webpack stuff in angular code.